.github/workflows/docker.yml

```diff
@@ -11,15 +11,27 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@v3
+
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v2
+
     - name: Login to Docker Hub
       uses: docker/login-action@v2
       with:
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_ACCESS_TOKEN }}
+
     - name: Build and push docker image
       uses: docker/build-push-action@v4
       with:
         push: true
         tags: getyourguide/istio-config-validator:latest
+
+    - name: Build and push istio-router-check
+      uses: docker/build-push-action@v4
+      with:
+        pull: true
+        file: tools/router-check-tool/Dockerfile
+        context: .
+        push: true
+        tags: getyourguide/istio-router-check:release-1.22
```

Comment:
Do we need to update the upgrade guide in relation to this?

Reply:
Yes, makes sense to automate this process. I intentionally left it out as for now it is only experimentation.